linode/manager
# M3-5244: Standardize secret token modals

## Description
Standardize secret token modals to one design (the tokens in the screenshot were deleted dw)

![image](https://user-images.githubusercontent.com/14323019/127165550-43d70c88-34a8-40ab-b9a9-0f0d3ae4a7d7.png)
![image](https://user-images.githubusercontent.com/14323019/127165588-5b90e2c6-a8bc-474e-85a5-0f5146be24ff.png)
![image](https://user-images.githubusercontent.com/14323019/127165617-0c20f111-a68e-45c2-afee-8bd1b4923c59.png)

## How to test
The new dialog should display a warning that the secret token will only be shown once. The keys/secret token should be in a copyable text field. The only way to dismiss the dialog is via the `x` button on the top right (clicking on the backdrop or pressing `esc` should not dismiss the dialog).
- Create a Personal Access Token in `profile/tokens`
- Create an OAuth App in `profile/clients`
- Create an Object Storage access key in `object-storage/access-keys`
